packages/oscal-react-library/src/components/OSCALEditableTextField.tsx

```diff
@@ -66,14 +85,38 @@ function textFieldWithEditableActions(props, reference, inEditState, setInEditSt
   );
 }
 
-export default function OSCALEditableTextField(props) {
+export interface EditableFieldProps {
+  isEditable?: boolean;
+  onFieldSave?: (...args: any[]) => void;
+  partialRestData?: any;
+}
+
+interface EditableTextFieldProps extends EditableFieldProps {
+  fieldName: string;
+  /**
+   * Whether the field is editable.
+   */
+  canEdit: boolean;
+  editedField: string[] | null;
+  /**
+   * The value that is edited?
+   */
+  editedValue?: any;
+  editedValueId?: string;
+  value?: string;
+  size?: number;
+  textFieldSize?: string;
+  typographyVariant?: TypographyVariant;
+}
```

Comment:
Fix comments

Reply:
Fixed them! The bottom 3 are hopefully self explanatory enough.